src/Devolutions.AvaloniaTheme.MacOS/Controls/ScrollViewer.axaml

```diff
@@ -96,7 +104,7 @@
     </Style>
     <Style Selector="^[AllowAutoHide=False]">
       <Style Selector="^ /template/ Border#PART_ScrollBarsSeparator">
-        <Setter Property="Opacity" Value="1" />
+        <Setter Property="Transitions" Value="{x:Null}" />
```

Comment:
Opacity is set above now. 

Removing the transition on page-load prevents false positives in the tests